shared/src/commonTest/kotlin/co/touchlab/kampkit/BreedModelTest.kt

```diff
@@ -31,9 +33,12 @@ class BreedModelTest : BaseTest() {
     private val settings = MockSettings()
     private val ktorApi = KtorApiMock()
 
+    // Need to start at non-zero time because the default value for db timestamp is 0
+    private val clock = ClockMock(Clock.System.now())
```

Comment:
I usually would still do a hardcoded number here rather than `now()`. But it's pretty unlikely to ever be an issue in this case, so no worries.